packages/grid_client/scripts/connection.ts

```diff
@@ -0,0 +1,90 @@
+/* 
+  Use this script to test grid connection:
+  This script creates 2 instances of the grid client, then after 10 minutes, it performs a query on both instances.
+
+  Steps:
+    1 - Connect to the grid
+    2 - Close the internet connection
+    3 - After 9 minutes, open the internet connection again
+  The grid should automatically reconnect and request the twin ID.
+*/
+
+import { GridClient, NetworkEnv } from "../src";
+
+// Replace with actual mnemonics
+const clientMnemonic1 = "your_mnemonic_here";
+const clientMnemonic2 = "your_mnemonic_here";
+const storeSecret = "store_secret";
+
+// Choose the desired network (main, test, qa, dev)
+const NETWORK = NetworkEnv.dev;
+
+// Function to log the current minute along with the client's name
+function logCurrentMinute(clientName) {
+  const now = new Date();
+  console.log(`${clientName} running in minute ${now.getMinutes()}`);
```

Comment:
logging the minute only looks strange, I think you should show the date in hour:min:sec

Reply:
Will remove the whole date and make count up instead